.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
```

Comment:
All the linter test are passed now, with the exception of the errors that state that some of the functions where used before defined, we can't do much about that because we need our gamerunner function to run at the end of the `script`. One that states that a function is assigned but never used, is being used in the HTML.